Sources/ImageCache.swift

```diff
@@ -93,6 +93,7 @@ open class ImageCache {
     
     /// The longest time duration in second of the cache being stored in disk. 
     /// Default is 1 week (60 * 60 * 24 * 7 seconds).
+    /// Setting this to a negative value will make the disk cache never expiring.
```

Comment:
It's intended to have only negative number for "forever". 0 means expiring immediately. And since there is a default non-zero value here, it is better to opt out 0 so we could avoid to introduce misunderstanding.